loaders/scss.js

```diff
@@ -22,7 +22,7 @@ module.exports = ({env, cwd, scssPath, useScssVariables, postcssPresetEnvOptions
             ${useScssVariables && `@import "variables";`}
           `,
           includePaths: [
-            path.join(cwd, scssPath || '/src/scss'),
+            path.join(cwd || process.cwd(), scssPath || '/src/scss'),
```

Comment:
We send the `cwd` from the top (and it defaults to `process.cwd()`). Why was this fallback added only here (since we have multiple places with `cwd` in there)?

Reply:
Let's put the defualt value in the function declaration and let's check for absolute paths
```js
path.isAbsolute(scssPath) ? scssPath : path.join(cwd, scssPath)
```